ci/telemetry_logging.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Telemetry Logging tests
+  hosts:  "{{ cifmw_target_hook_host | default('localhost')  }}"
+  gather_facts: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars_files:
+    - vars/common.yml
+    - vars/osp18_env.yml
+  tasks:
+    - name: "Run logging Tests"
+      ansible.builtin.import_role:
+        name: telemetry_logging
+
+        - import_role:
```

Comment:
This task in not well formatted, and needs to be removed

Reply:
replaces with run_playbooks_logging.yml